oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnection.java

```diff
@@ -135,8 +180,11 @@ public boolean isAvailable() {
 
     @Override
     public synchronized void close() throws IOException {
-        if (client != null) {
-            client.close();
+        if (esClient != null) {
+        	esClient._transport().close();
```

Comment:
is esAsyncClient also closed as part of this call ? Or should we close that separately ? Since it's created independently - I would assume it would need to be closed separately too (but I might be wrong here.)

Reply:
Both clients are based on the same transport layer. We just need to close one. Added a comment to clarify that.